db/schema.rb

```diff
@@ -10,13 +10,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_07_08_112144) do
+ActiveRecord::Schema.define(version: 2021_07_14_132606) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
-  create_table "activities", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+  create_table "activities", id: :uuid, default: -> { "public.gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
Is this addition of a `public` prefix? namespace? necessary, or unrelated artefact of local dev stuff?

Reply:
Same. This should have been stripped out by me, but got caught up in rebase hell